Models/Shape.cs

```diff
@@ -11,8 +11,36 @@ namespace DrawMat.Models;
 
 public abstract class ShapeBase
 {
-    public abstract Control ToControl();
+    public Rect BoundingBox { get; set; } = new();
```

Comment:
Since the bounding box is a derived property, there shouldn't be a public setter.